backend/subroker/views.py

```diff
@@ -8,8 +8,9 @@
 import json
 from json.decoder import JSONDecodeError
 from .models import Ott, Group, Content, Review
-from django.core.exceptions import ObjectDoesNotExist
 from django.db.models import Q
+import pip._vendor.requests as requests
```

Comment:
@muhwagwa how is it different from import requests?

Reply:
@arclic I had some problem importing 'requests' and found this solution on stackoverflow! If it doesn't cause any issues for you, it's fine to just import requests